Blackbaud.AppFx.MajorGiving.Mobile.Sky/Crm.Frog.Ui.nuspec

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
-        <id>Crm.Frog.Ui</id>
+        <id>Crm.Frog.Ui.DotNetMigration</id>
```

Comment:
I understand that this was a temporary change to differentiate between packages, but I don't think we want to merge this to `master`. If that's correct, please roll this back in `dot-net-migration` so it is removed from the merge.

Reply:
This file can be removed from the Merge - teh one in master is fine :)